assets/scss/_grpc.scss

```diff
@@ -445,3 +445,9 @@ c - Component (Aware of its content/context...)
     display: none;
   }
 }
+
+.youtube-video iframe {
+  width: 560px;
```

Comment:
This doesn't look good on mobile. It runs off the screen. Can we fit the width to the parent container instead? In general, we should be testing all CSS changes [in mobile configuration under developer tools](https://developer.chrome.com/docs/devtools/device-mode).

Reply:
![BuftpkDx5ayJP3m](https://github.com/user-attachments/assets/8bc388cf-1b4a-4fe3-9a0d-62f364bb058a)

Reply:
added an aspect ratio of 46% which is good for small, medium and large screens.

Reply:
Looks good! Thanks!